client/src/pages/CreateCluster.tsx

```diff
@@ -40,9 +40,9 @@ const SmallHeading = styled(SubTitle)`
 const ClusterBox = styled.div`
   width: 600px;
   height: 300px;
```

Comment:
This doesn't need to happen in this PR but widths and heights should be in rem, not px